types.go

```diff
@@ -153,3 +157,14 @@ type Currency struct {
 	Precision int
 	Symbol    string
 }
+
+// RTAUStatus represents the Real Time Account Updater response from a processor, if applicable
+type RTAUStatus string
+
+const (
+	RTAUStatusUnknown      RTAUStatus = "Unknown"
+	RTAUStatusCardChanged  RTAUStatus = "CardChanged"
+	RTAUStatusCardExpired  RTAUStatus = "CardExpiryChanged"
+	RTAUStatusCloseAccount RTAUStatus = "CloseAccount"
+	RTAUExpiryTimeFormat = "1/2006"
```

Comment:
Let's have this outside of the const block. Ususally when we defined a const block after declaring a type like `RTAUStatus`, we enumerate the constants of that type. The RTAUExpiryTimeFormat is separate.